integration_tests/elasticsearch-sink/create_sink.sql

```diff
@@ -14,6 +15,7 @@ CREATE SINK bhv_es8_sink
 FROM
     bhv_mv WITH (
     connector = 'elasticsearch',
+    type = 'upsert',
```

Comment:
Is the case of partial update already covered in the current tests? If not we should add some tests to cover it.